tests/Speedtest/PerformanceReport/2024_3_15_PerformanceReport_Combined.md

```diff
@@ -0,0 +1,12 @@
+Measured on 13th Gen Intel(R) Core(TM) i7-13800H
+
```

Comment:
do we want to push all of these? These are tracked by git history.

Reply:
if we want to track them remove date created from performance report filename and add it as content to the file.

Reply:
Definitely want to track this. 

You mean put the different reports over time into a single file like with release_notes.md

Reply:
or simply replace them always, because we have git history to track them over time. And not have one report per language per run

Reply:
I see your point. In this case though, I think an easy comparison of the different states over time might be beneficial, which is not the strength of the git history.

Reply:
Will always replace the single reports now. Combined report is an infinite list like Release_notes.md now